package.json

```
@@ -17,7 +17,7 @@
     "coverage": "nyc -r html npm test"
   },
   "dependencies": {
-    "ip": "^1.1.5"
+    "@leichtgewicht/ip-codec": "^2.0.1"
```

```suggestion
    "@leichtgewicht/ip-codec": "2.0.1"
```

From a security standpoint, it's better to depend on exact version.

---

There are different schools of thought on this, I just kept it as it was before.